fec/models.py

```diff
@@ -26,92 +28,90 @@ def party_abbreviation(self):
     def party_full(self):
         return self.PARTIES.get(self.party, self.party)
 
-    def similar_campaigns(self):
-        shared_contributors = Count(
-            'committee__contribution__contributor',
-            distinct=True,
-            filter=Q(
-                committee__contribution__contributor__contribution__committee__campaign__id
-                =self.id))
+    def similar_candidates(self):
+        connections = self.source_connections.order_by(
+            "-score").select_related("target")[:10]
 
-        campaigns = Campaign.objects.exclude(pk=self.pk).annotate(
-            contributor_count=shared_contributors).exclude(
-                contributor_count=0).order_by("-contributor_count")
-        return campaigns[:10]
+        candidates = []
+        for connection in connections:
+            target = connection.target
+            target.contributor_count = connection.score
+            candidates.append(target)
+
+        return candidates
+
+    @staticmethod
+    def connected_candidates():
+        return Candidate.objects.exclude(source_connections=None)
 
     @staticmethod
     def search(data):
         """Search by name, id, or comittee id."""
 
         # Note the distinct(), because the many-to-many committee__id
         # check returns duplicates.
-        return Campaign.objects.filter(
+        return Candidate.objects.filter(
             Q(id=data) | Q(name__icontains=data)
-            | Q(committee__id=data)).distinct()
+            | Q(committee__committee_id=data)).distinct()
 
 
 class Committee(models.Model):
-    id = models.CharField(max_length=9, primary_key=True)
-
-    campaign = models.ForeignKey(
-        Campaign, on_delete=models.PROTECT, blank=True, null=True)
-
-    name = models.CharField(max_length=200)
-
-
-class Contributor(models.Model):
```

Comment:
At least for now, we don't need a contributor model anymore. It's all still in the git history for if we want to reference it later though